modules/spock/src/main/groovy/org/testcontainers/spock/SpockTestDescription.groovy

```diff
@@ -0,0 +1,36 @@
+package org.testcontainers.spock
+
+import groovy.transform.PackageScope
+import org.spockframework.runtime.extension.IMethodInvocation
+import org.testcontainers.lifecycle.TestDescription
+
+/**
+ * Spock specific implementation of a Testcontainers TestDescription.
+ *
+ * Filesystem friendly name is based on Specification and Feature.
+ */
+@PackageScope
+class SpockTestDescription implements TestDescription {
+
+    String specName
+    String featureName
+
+    static SpockTestDescription fromTestDescription(IMethodInvocation invocation) {
+        return new SpockTestDescription([
+            specName: invocation.spec.name,
+            featureName: invocation.feature.name
```

Comment:
FYI `invocation.feature` is marked as `@Nullable`

Reply:
probably `getMethod()` is more appropriate here?

Reply:
k, will check in the implementation.

Reply:
Actually I don't understand the null condition from the javadoc:
```
   * Returns the feature which this method invocation belongs to (if any).
   * Differs from {@code MethodInfo.getFeature()} in that it reflects the dynamic
   * picture. For example, when a setup method is invoked, this method
   * will return the corresponding feature, whereas {@code MethodInfo.getFeature()}
   * will return {@code null}.
```

Reply:
So if potentially called from within a `setup` block, we would still get the actual feature name, which is good IMO.

So TBH, I'd prefer to keep feature how, although I still wonder how it might be `nullabe`.

Reply:
So FY, in the context of our extension execution `getMethod()` is actually `null`, so let's keep like this for now :man_shrugging:

Reply:
Ok, I trust you here :)